dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/EditDatasetMetadataPage.java

```diff
@@ -100,7 +106,10 @@ public String init() {
         if (!permissionsWrapper.canUpdateDataset(dvRequestService.getDataverseRequest(), dataset)) {
             return permissionsWrapper.notAuthorized();
         }
-        if (!dataset.getLocks().isEmpty()) {
+        /**
+         * Locked dataset can be edited if it's InReview by a user with edit and publish permissions
```

Comment:
I think the code is self-explanatory, I would remove it. I think Javadocs (on methods) should be used mainly for explaining the context of invoking the method, not the implementation itself.